exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Table.kt

```diff
@@ -918,13 +918,13 @@ open class Table(name: String = "") : ColumnSet(), DdlAware {
     // Column references
 
     /**
-     * Create reference from a @receiver column to [ref] column.
+     * Creates a reference from this @receiver column to a [ref] column.
      *
-     * It's a short infix version of [references] function with default onDelete and onUpdate behavior.
+     * This is a short infix version of `references()` with default `onDelete` and `onUpdate` behavior.
      *
-     * @receiver A column from current table where reference values will be stored
+     * @receiver A column from the current table where reference values will be stored.
      * @param ref A column from another table which will be used as a "parent".
-     * @see [references]
+     * @sample org.jetbrains.exposed.sql.tests.shared.dml.JoinTests.testJoin04
```

Comment:
This is supposed to be a link to an overload function, but it goes nowhere because of [KT-15984](https://youtrack.jetbrains.com/issue/KT-15984).